application/gqlgen/schema.graphql

```diff
@@ -178,6 +177,12 @@ type Mutation {
     "Cancel a carry offer for any user. Authorized for the post creator, the potential provider, and Super Admins."
     removePotentialProvider(postID: String!, userID: String!): Post!
 
+    "Provider changes the status of a request to DELIVERED"
+    markRequestAsDelivered(postID: String!): Post!
+
+    "Requester changes the status of a request to RECEIVED"
+    markRequestAsReceived(postID: String!): Post!
+
```

Comment:
Are these effectively shortcuts or aliases to `updatePostStatus`? I'm not sure what this gains us.

Reply:
Matt was asking for them so that the UI doesn't need to know what goes on in the backend.  For example, for now a Post that is Accepted or Delivered goes straight to Completed if the creator says "I received it".  Some day, though, it may just go to Received until another step has taken place, e.g. the provider says that they got paid.

Reply:
Still seems like a status update with an internal rule that says some states can "auto-advance" to the next state. That would be reflected on the value in the `status` field in the response.